app.mjs

```diff
@@ -29,6 +30,21 @@ const chainIdParamSchema = {
  * @param {import("fastify").FastifyInstance} fastify
  */
 export async function server(fastify, opts) {
+  Sentry.init({
```

Comment:
Also, can we make this optional? We don't necessarily want sentry running  in a dev environment